rs/registry/canister/canister/canister.rs

```diff
@@ -186,19 +186,31 @@ fn canister_init() {
         println!("{LOG_PREFIX}canister_init: Overriding swapping flags");
         println!(
             "{LOG_PREFIX}canister_intt: Swapping enabled: {}",
-            init_payload.is_swapping_feature_enabled
+            init_payload.is_swapping_feature_enabled.unwrap_or(false)
         );
-        test_set_swapping_status(init_payload.is_swapping_feature_enabled);
+        test_set_swapping_status(init_payload.is_swapping_feature_enabled.unwrap_or(false));
         println!(
             "{LOG_PREFIX}canister_init: Swapping whietlisted callers: {:?}",
-            init_payload.swapping_whitelisted_callers
+            init_payload
+                .swapping_whitelisted_callers
+                .clone()
+                .unwrap_or(Vec::new()),
```

Comment:
I think the original code here works. Going back avoids (unnecessary) cloning, but it's not a big savings, so if you can't make it work immediately, just stick with this so we can get moving.